libs/mrwebrtc/src/pch.h

```diff
@@ -34,6 +33,12 @@
 
 #endif
 
+#if (__cplusplus >= 201703L)
```

Comment:
Make sure that we are using https://docs.microsoft.com/en-us/cpp/build/reference/zc-cplusplus

Reply:
Fixed.